scripts/testing/tolerance_test.py

```diff
@@ -1,78 +1,221 @@
+#!/bin/sh
+"exec" "python3" "-u" "-B" "$0" "$@"
+##############################################################################
+# Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+# other Serac Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+##############################################################################
+
+import argparse
 import json
-import argparse # for command line arguments
-import math # for comparing values within tolerance
-
- # l2norms last number in last temp listed changed from 9 to 8 (in small_change)
- # mins first number in second temp listed changed from 1 to 100 (big_change)
- 
- # load name of each field from file & populate them in a list 
-def get_field_names(d):    
+import math
+import os
+import sys
+
+
+# This script compares two Serac summary files against each other.
+# The 'baseline' file is treated as a truth against the possibly wrong
+# 'test' file. It attempts to fail with the most error messages it can at a given
+# correctness level (field names > sample names > sample values).
+# Summary files were the same within the given tolerance if the
+# script exists successfully and not otherwise.
```

Comment:
regarding alternate names: I think of the {norms, mean, min, max} as "stats" / "statistics", "features", or "properties" of a given field

Reply:
I like "stats" or "metrics".

Reply:
went with 'stats'